docs/template_extension/new/ov_extension.cpp

```diff
@@ -4,14 +4,16 @@
 
 #include <openvino/core/extension.hpp>
 #include <openvino/core/op_extension.hpp>
+#include <openvino/frontend/extension.hpp>
 
 #include "identity.hpp"
 
 // clang-format off
 //! [ov_extension:entry_point]
 OPENVINO_CREATE_EXTENSIONS(
     std::vector<ov::Extension::Ptr>({
-        std::make_shared<ov::OpExtension<TemplateExtension::Identity>>()
+        std::make_shared<ov::OpExtension<TemplateExtension::Identity>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::Identity>>("TemplateIdentity")
```

Comment:
Why do you map to "TemplateIdentity" not "Identity"?

Reply:
To visually distinguish original FW op name from our op name. That doesn't really matter, and now I see that probably just "Identity" is good enough.